tmt/steps/provision/__init__.py

```diff
@@ -146,3 +150,231 @@ def guest(self):
         Should return a provisioned Guest() instance.
         """
         raise NotImplementedError
+
+
+class Guest(tmt.utils.Common):
+    """
+    Guest environment provisioned for test execution
```

Comment:
_Note:_ not to be confused with actual `ENV` which is passed directly to `execute`.

Reply:
Ok, removed to prevent confusion.